checks/checkforcontent.go

```diff
@@ -78,10 +87,18 @@ func CheckFilesContent(checkName, shellPathFnPattern string, c *checker.CheckReq
 			continue
 		}
 
-		name := names[1]
-		// Filter out files based on path.
-		if match, _ := path.Match(shellPathFnPattern, name); !match {
-			continue
+		fullpath := names[1]
+		filename := path.Base(fullpath)
+
+		if !caseSensitive {
+			fullpath = strings.ToLower(fullpath)
+			filename = strings.ToLower(path.Base(fullpath))
```

Comment:
maybe move this line before fullpath is lowered in case, before line 94.

filename = strings.ToLower(path.Base(fullpath))
fullpath = strings.ToLower(fullpath)

Reply:
I can, but what's the reasoning?
I think we could change it to:
```
fullpath = strings.ToLower(fullpath)			
filename = path.Base(fullpath)
```
to avoid having ToLower iterate over the string again.

Reply:
yes skip, i thought path.Base was dynamic, but it just gets a section of path.